README.md

```diff
@@ -1,10 +1,102 @@
 # kv-go
-Distributed in-memory (at least for now) KV storage written in Go
 
-#Motivation
-Just For Fun
-This is an educational project about how to develop scalable distributed systems.
-
-#Disclaimer
+# Disclaimer
 Use this project at your own risk.
```

Comment:
It's better to mark explicitly that it's an educational purpose system, no practical application is meant at all.